examples/otel-logs-with-sourcetypes-splunk/otel-collector-config.yml

```diff
@@ -0,0 +1,72 @@
+receivers:
+    filelog/onefile:
+      include: [ /output/file.log ]
+    filelog/twofile:
+      include: [ /output/file2.log ]
+    filelog/threefolder:
+      include: [ /output3/*.log ]
+
+exporters:
+    splunk_hec/logs:
+        # Splunk HTTP Event Collector token.
+        token: "00000000-0000-0000-0000-0000000000000"
+        # URL to a Splunk instance to send data to.
+        endpoint: "https://splunk:8088/services/collector"
+        # Optional Splunk source: https://docs.splunk.com/Splexicon:Source
+        source: "output"
+        # Default sourcetype. This example overrides sourcetype, so it will never be used.
+        sourcetype: "otel:shouldnotshow"
```

Comment:
Does this mean that sourcetype is a required field? Perhaps we should add a comment indicating either that it is, or why we are setting it here.

Reply:
If we don't set it, Splunk Enterprise sets a default value of its own instead based on the token configuration. We can remove it I think to avoid going into that conversation.